doc/api/async_hooks.md

```diff
@@ -462,6 +464,8 @@ will occur and node will abort.
 The following is an overview of the `AsyncResource` API.
 
 ```js
+const { AsyncResource } = require('async_hooks');
```

Comment:
This is the only place where a reader can find out how to obtain `AsyncResource`.